File: lab/Dev c++/-12.cpp
struct carstruct {
	char Brand[30],Gen[30],Color[30];
	int Price;
}car[30];
#include<stdio.h>
int main(){
	int n,i;
	printf("numbar car = ");
	scanf("%d",&n);
	while (n <= 0){
		printf("number car = ");
		scanf("%d",&n);
	}
	for(i = 0; i < n; i = i + 1){
		printf("carBrand = ");
		scanf("%s",&car[i].Brand);
		printf("carGen = ");
		scanf("%s",&car[i].Gen);
		printf("carColor = ");
		scanf("%s",&car[i].Color);
		printf("carPrice = ");
		scanf("%d",&car[i].Price);
	}
	for (i = 0; i < n; i = i + 1){
		printf("carBrand = %s \n",car[i].Brand);
		printf("carGen = %s \n",car[i].Gen);
		printf("carColor = %s \n",car[i].Color);
		printf("carPrice = %d \n",car[i].Price);
		
	}
}

File: lab/ข้อ 12/-12.py
Brand = []
Gan = []
Color = []
Price = []
n = int(input("จำนวนข้อมูล = "))
while n <= 0:
    n =int(input('จำนวนรถยนต์ = '))
for i in range(n):
    Brand.append(str(input('ยีห้อรถยนต์ = ')))
    Gan.append(str(input('รุ่นของรถยนต์ = ')))
    Color.append(str(input('สีของรถยนต์ = ')))
    Price.append(str(input('ราคารถยนต์ = ')))
    car = {
        'Brand': Brand,
        'Gen': Gan,
        'Color': Color,
        'Price': Price,
    }
print('ยีห้อรถยนต์ =',car['Brand'])
print('รุ่นของรถยนต์ =',car['Gen'])
print('สีของรถยนต์',car['Color'])
print('ราคารถยนต์ =',car['Price'])
